themes/src/main/resources/theme/keycloak-preview/account/index.ftl

```diff
@@ -109,6 +109,7 @@
                 } else {
                     document.getElementById("landingSignOutButton").style.display='inline';
                     document.getElementById("landingSignOutLink").style.display='inline';
+                    document.getElementById("loggedInUser").innerHTML = keycloak.tokenParsed.preferred_username;
```

Comment:
Does this mean Welcome Page doesn't display first and last name of a user?

Reply:
It is the same logic only the username

Reply:
done